packages/config/config/devices/0x0344/he-tps05.json

```diff
@@ -0,0 +1,970 @@
+{
+	"manufacturer": "HELTUN",
+	"manufacturerId": "0x0344",
+	"label": "HE-TPS05",
+	"description": "Touch Panel Switch Quinto",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x0003"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Frequency Region",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 9,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "EU (868.4 MHz, 869.85 MHz)",
+					"value": 0
+				},
+				{
+					"label": "US (908.4 MHz, 916.0 MHz)",
+					"value": 1
+				},
+				{
+					"label": "AU (919.8 MHz, 921.4 MHz)",
+					"value": 2
+				},
+				{
+					"label": "HO (919.8 MHz)",
+					"value": 3
+				},
+				{
+					"label": "In (865.2 MHz)",
+					"value": 4
+				},
+				{
+					"label": "IL (916.0 MHz)",
+					"value": 5
+				},
+				{
+					"label": "RU (869.0 MHz)",
+					"value": 6
+				},
+				{
+					"label": "Cn (868.4 MHz)",
+					"value": 7
+				},
+				{
+					"label": "JP (922.5 MHz, 923.9 MHz, 926.3 MHz)",
+					"value": 8
+				},
+				{
+					"label": "Or (920.9 MHz, 921.7 MHz, 923.1 MHz)",
+					"value": 9
+				}
+			]
+		},
+		"3": {
+			"label": "Hardware and Software Versions",
+			"valueSize": 4,
+			"minValue": 0,
+			"maxValue": 0,
+			"readOnly": true
```

Comment:
None of these can do anything with a min of 0 and a max of 0 even if readonly. There are a couple of them.

Reply:
Please advise on what to put there. Since it's readonly, min and max does not have any sense, though it's requested.

Reply:
Even if it's read only there is still a range of values possible. Most manufacturers still list the min and max. 

@AlCalzone we still need to provide a range for the UIs, right? This isn't a candidate to remove for read only params?

Reply:
I've thought about it... Applications may use the information to show a gauge or something but sometimes we really don't know the range. If this isn't an error, we could leave it at 0/0 IMO.